### PR TITLE
[libc][docs] add page linking to talks

### PR DESCRIPTION
**Pull request by nickdesaulniers**

### CODE REVIEW

**Review by jhuber6 — APPROVED** [review]
Ah, thanks for this.

**Review by michaelrj-google — APPROVED** [review]
overall LGTM

File: libc/docs/talks.rst [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  * `slides <https://llvm.org/devmtg/2023-10/slides/techtalks/Huber-LibCforGPUs.pdf>`__
+  * `video <https://www.youtube.com/watch?v=_LLGc48GYHc>`__
+
+----
+2022
+----
+* Using LLVM's libc - Sivachandra Reddy, Michael Jones, Tue Ly
+
+  * `slides <https://llvm.org/devmtg/2022-11/slides/Tutorial1-UsingLLVM-libc.pdf>`__
+  * `video <https://www.youtube.com/watch?v>`__
```
[the remark this answers is not in the document]
Comment (nickdesaulniers) [reply libc/docs/talks.rst]: good catch, fixed in https://github.com/llvm/llvm-project/pull/84393/commits/532831eeba055a49a8f05f7905987055adbc3417

**Comment by nickdesaulniers** [comment]
FWIW, I had to use double underscores for the hyperlink syntax due to: https://jwodder.github.io/kbits/posts/rst-hyperlinks/#gotcha

File: libc/docs/talks.rst [hunk 2]
```diff
@@ -0,0 +1,29 @@
+=====
+Talks
+=====
+----
+2023
+----
+* Math functions in LLVM libc or yet another correctly rounded libm - Tue Ly
+
+  * `video <https://www.youtube.com/watch?v=kBSJqVWNQLY>`__
```
Comment (nickdesaulniers) [inline libc/docs/talks.rst:9]: @lntue I couldn't find a link to publicly accessible slides; do you have that? Mind adding it to this page, or sharing the link with me and I will add?

Comment (lntue) [reply libc/docs/talks.rst:9]: Does the link https://drive.google.com/file/d/1Gchrc3AXKXAWFI1nk8Y1x8UsoQSp8XjF/view from https://fpbench.org/talks/fptalks23.html work?

Comment (nickdesaulniers) [reply libc/docs/talks.rst:9]: Yes; https://github.com/llvm/llvm-project/pull/84516

**Pull request merged**